src/verifai/features/features.py

```diff
@@ -1074,6 +1103,39 @@ def flatten(self, point, fixedDimension=False):
                         flattened.append(None)
             else:
                 domain.flattenOnto(value, flattened)
+
+        flattened.append(len(point.dynamicSampleHistory))
```

Comment:
If the feature space does not use time series at all, can we omit this? Then the flattened vectors won't change for existing code, which is nice for backwards compatibility as well as eliminating a redundant column in the error table. (You'd have to remove the +1 in `fixedFlattenedDimension` too in this case.)